src/tools/ceph_monstore_tool.cc

```diff
@@ -620,7 +622,39 @@ int main(int argc, char **argv) {
                 << " version " << v << " to " << outpath
                 << std::endl;
     }
-  } else if (cmd == "dump-paxos") {
+ }else if (cmd == "list-version") {
```

Comment:
--versions or --show-versions?  --list-version suggests to me that it'll list out stored versions (and not the ends of the range).